enterprise/frontend/src/metabase-enterprise/plugins.js

```diff
@@ -18,6 +18,7 @@ import "./database_routing";
 import "./whitelabel";
 import "./embedding";
 import "./embedding-sdk";
+import "./embedding_iframe_sdk";
```

Comment:
i'm not convinced about the case. I think this should be kebab-case following other siblings here.

Reply:
@WiNloSt actually `embedding-sdk` is wrong, all other EE plugin is `underscore_case` not `kebab-case`

![CleanShot 2568-05-09 at 23 05 07@2x](https://github.com/user-attachments/assets/2bbda834-7e08-41a2-80fc-471cd9f4e7fc)

Reply:
You're absolutely right. Just keep it as is.

Reply:
I created an issue for that https://linear.app/metabase/issue/EMB-383/make-ee-plugin-directory-name-cases-consistent